config/rapidez/statamic.php

```diff
@@ -80,13 +80,10 @@
 
     'sites' => [
         'default' => [
-            'name' => env('APP_NAME', 'Statamic'),
             'locale' => 'en_EN',
             'lang' => 'en_EN',
             'url' => '/',
             'attributes' => [
-                'magento_store_id' => 1,
-                'group' => 'default',
                 'disabled' => false,
             ],
```

Comment:
Do we still need this whole sites config?

Reply:
Changed this around, see https://github.com/rapidez/statamic/pull/81#issuecomment-2457255941